packages/cli/src/controllers/passwordReset.controller.ts

```diff
@@ -239,21 +266,42 @@ export class PasswordResetController {
 
 		const validPassword = validatePassword(password);
 
-		// Timestamp is saved in seconds
-		const currentTimestamp = Math.floor(Date.now() / 1000);
+		let decodedToken: jwt.JwtPayload;
+
+		try {
+			decodedToken = jwt.verify(resetPasswordToken, config.getEnv('userManagement.jwtSecret'), {
```

Comment:
it looks like we are duplicating this code. maybe we should create a `services/jwt.service.ts` to abstract this code away, and add unit tests for that service as well.

Reply:
Moved that that logic into a private method in the same controller and create a jwtService to wrap the library